client-templates/gitlab/accept.json.sample

```diff
@@ -51,108 +51,216 @@
+      "path": "/api/v4/projects/:project/repository/files*%2Fpackages.config",
+      "origin": "https://${GITLAB}"
+    },
     {
       "//": "used to determine the full dependency tree",
       "method": "GET",
       "path": "/api/v4/projects/:project/repository/files*/*.csproj",
       "origin": "https://${GITLAB}"
     },
+    {
+      "//": "used to determine the full dependency tree",
+      "method": "GET",
+      "path": "/api/v4/projects/:project/repository/files*%2F*.csproj",
+      "origin": "https://${GITLAB}"
+    },
     {
       "//": "used to determine the full dependency tree",
       "method": "GET",
       "path": "/api/v4/projects/:project/repository/files*/*.vbproj",
       "origin": "https://${GITLAB}"
     },
+    {
+      "//": "used to determine the full dependency tree",
+      "method": "GET",
+      "path": "/api/v4/projects/:project/repository/files*%2F*.vbproj",
+      "origin": "https://${GITLAB}"
+    },
     {
       "//": "used to determine the full dependency tree",
       "method": "GET",
       "path": "/api/v4/projects/:project/repository/files*/*.fsproj",
       "origin": "https://${GITLAB}"
     },
+    {
+      "//": "used to determine the full dependency tree",
+      "method": "GET",
+      "path": "/api/v4/projects/:project/repository/files*%2F*.fsproj",
+      "origin": "https://${GITLAB}"
+    },
     {
       "//": "used to determine the full dependency tree",
       "method": "GET",
       "path": "/api/v4/projects/:project/repository/files*/project.json",
       "origin": "https://${GITLAB}"
     },
+    {
+      "//": "used to determine the full dependency tree",
+      "method": "GET",
+      "path": "/api/v4/projects/:project/repository/files*%2Fproject.json",
+      "origin": "https://${GITLAB}"
+    },
     {
       "//": "used to determine the full dependency tree",
       "method": "GET",
       "path": "/api/v4/projects/:project/repository/files*/Gopkg.toml",
       "origin": "https://${GITLAB}"
     },
+    {
+      "//": "used to determine the full dependency tree",
+      "method": "GET",
+      "path": "/api/v4/projects/:project/repository/files*%2FGopkg.toml",
+      "origin": "https://${GITLAB}"
+    },
     {
       "//": "used to determine the full dependency tree",
       "method": "GET",
       "path": "/api/v4/projects/:project/repository/files*/Gopkg.lock",
       "origin": "https://${GITLAB}"
     },
+    {
+      "//": "used to determine the full dependency tree",
+      "method": "GET",
+      "path": "/api/v4/projects/:project/repository/files*%2FGopkg.lock",
+      "origin": "https://${GITLAB}"
+    },
     {
       "//": "used to determine the full dependency tree",
       "method": "GET",
       "path": "/api/v4/projects/:project/repository/files*/vendor.json",
       "origin": "https://${GITLAB}"
     },
+    {
+      "//": "used to determine the full dependency tree",
+      "method": "GET",
+      "path": "/api/v4/projects/:project/repository/files*%2Fvendor.json",
```

Comment:
And here we allow any `vendor.json` file, not just `/vendor/vendor.json`. Outside of the scope of this PR, but needs to be aligned.